src/tasks/helpers/base.ts

```diff
@@ -78,6 +78,10 @@ export abstract class TaskHelper {
 		builder.addOption('--target', definition.target as string);
 
 		builder.addQuotedOption('--sdk', project.sdk()[0]);
+
+		if (definition.extraArguments) {
```

Comment:
I think this will need to be performed in `resolveCommonOptions` so that it works for both module and app projects